frontend/src/__tests__/storagCreationValidation.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { canCreate, canAttachExisting } from "../old-pages/Configure/Storage";
+
+
+describe("Given a function to determine whether we can create a storage of a given type", () => {
+    describe("when the storage type is null", () => {
+        it("should return false", () => {
```

Comment:
suggestion: it's better to avoid direct references to the implementation in the test spec to do a proper decoupling.
In this case "should return false" may be rephrased as "should not allow the creation of a new storage". Same goes for "when the storage type is null". 

Overall great tests!